### PR TITLE
add new icons to icon font, search on small rezolution

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: themes/qgrid/layout/partials/header.ejs [hunk 1]
```diff
@@ -5,6 +5,8 @@
 <% } else { %>
 <button class="nav-trigger" id="nav-trigger"></button>
 <a href="<%- url_for('/') %>" class="logo"></a>
+<button class="search-trigger" id="search-trigger"></button>
+<input id="search" class="search" placeholder="Search"/>
```
Comment (klumba12) [inline themes/qgrid/layout/partials/header.ejs]: type="search"

File: themes/qgrid/source/css/nav.scss [hunk 2]
```diff
@@ -16,6 +16,10 @@
     height: 69px;
     border-bottom: 1px solid white;
     display: none;
+
+    .search {
+      width: calc(100% - 34px);
```
Comment (klumba12) [inline themes/qgrid/source/css/nav.scss]: can we apply margin/padding instead of calc?

File: themes/qgrid/source/js/menu.js [hunk 3]
```diff
@@ -32,21 +32,35 @@ function filter(e) {
 	}
 }
 
+function showSearch(e) {
+	var target = e.target || e.srcElement;
+	target.style.display = "none";
+	var search = document.getElementById('search');
+	console.log(search);
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: remove console

File: themes/qgrid/source/js/menu.js [hunk 4]
```diff
@@ -32,21 +32,35 @@ function filter(e) {
 	}
 }
 
+function showSearch(e) {
+	var target = e.target || e.srcElement;
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: here `target` doesn't say what should be hidden, is it trigger button? can we just assing appropriate class like '.active' and hide button by css? and show search by css too

File: themes/qgrid/source/css/nav.scss (hunk 2 above)
Comment (etokareva) [reply themes/qgrid/source/css/nav.scss]: I have to stretch this input

File: themes/qgrid/layout/partials/header.ejs (hunk 1 above)
Comment (etokareva) [reply themes/qgrid/layout/partials/header.ejs]: I don't like default search type styles

Comment (etokareva) [reply themes/qgrid/layout/partials/header.ejs]: I don't like defauld Chrome and Safari styles for this type of input

**Review by klumba12 — APPROVED** [review]

**Pull request merged**